signac/contrib/job.py

```diff
@@ -486,6 +492,8 @@ def init(self, force=False):
 
         Returns the calling job.
 
+        See :ref:`signac-cli-job` ``--init`` for the command line equivalent.
```

Comment:
Looks like we named init/create differently between the CLI and Python API.
```suggestion
        See :ref:`signac job -c <signac-cli-job>` for the command line equivalent.
```